api/mesh/v1alpha1/gateway.proto

```diff
@@ -29,7 +29,6 @@ message MeshGateway {
   option (kuma.mesh.resource).name = "MeshGatewayResource";
   option (kuma.mesh.resource).type = "MeshGateway";
   option (kuma.mesh.resource).package = "mesh";
-  option (kuma.mesh.resource).is_experimental = true;
```

Comment:
What's the effect of this, why is it in this PR?

Reply:
I did not notice this, good catch - @lahabana ?